godeltaprof/compat/stub_go23_test.go

```diff
@@ -0,0 +1,42 @@
+//go:build go1.23
+// +build go1.23
+
+package compat
+
+import "testing"
+
+func TestRuntimeFrameSymbolName(t *testing.T) {
+	checkSignature(t, "runtime/pprof",
+		"runtime_FrameSymbolName",
+		"func runtime/pprof.runtime_FrameSymbolName(f *runtime.Frame) string")
+	checkSignature(t, "github.com/grafana/pyroscope-go/godeltaprof/internal/pprof",
+		"runtime_FrameSymbolName",
+		"func github.com/grafana/pyroscope-go/godeltaprof/internal/pprof.runtime_FrameSymbolName(f *runtime.Frame) string")
+}
+
+func TestRuntimeFrameStartLine(t *testing.T) {
+	checkSignature(t, "runtime/pprof",
+		"runtime_FrameStartLine",
+		"func runtime/pprof.runtime_FrameStartLine(f *runtime.Frame) int")
+	checkSignature(t, "github.com/grafana/pyroscope-go/godeltaprof/internal/pprof",
+		"runtime_FrameStartLine",
+		"func github.com/grafana/pyroscope-go/godeltaprof/internal/pprof.runtime_FrameStartLine(f *runtime.Frame) int")
+}
+
+func TestRuntimeExpandFinalInlineFrame(t *testing.T) {
+	checkSignature(t, "runtime/pprof",
+		"runtime_expandFinalInlineFrame",
+		"func runtime/pprof.runtime_expandFinalInlineFrame(stk []uintptr) []uintptr")
+	checkSignature(t, "github.com/grafana/pyroscope-go/godeltaprof/internal/pprof",
+		"runtime_expandFinalInlineFrame",
+		"func github.com/grafana/pyroscope-go/godeltaprof/internal/pprof.runtime_expandFinalInlineFrame(stk []uintptr) []uintptr")
+}
+
+func TestRuntimeCyclesPerSecond(t *testing.T) {
```

Comment:
I think you forgot to remove this test:

https://github.com/grafana/pyroscope-go/blob/ffd26151ec3a9274af9dc11ff96e42bcd249a062/godeltaprof/compat/stub_test.go#L25-L29

Reply:
Actually I think all of the tests in `stub_tests.go` are duplicated by the version-specific tests. So I removed all tests from  `stub_test.go`.